README.md

```diff
@@ -108,167 +87,67 @@ There are two ways to authenticate:
- ▸    Hint: Test this app's login box with 'auth0 test login wmVzrZkGhKgglMRMvpauORCulBkQ5qeI'
- ▸    Hint: You might wanna try 'auth0 quickstarts download wmVzrZkGhKgglMRMvpauORCulBkQ5qeI'
-```
-
-As you might observe, the next thing to do would likely be to try logging in
-using the client ID.
-
-### Test the login flow
-
-Whether or not you've created the application using the CLI or the management
-dashboard, you'll be able to test logging in using a specific application.
-
-If you have the client ID, you may pass it as an argument,
-otherwise a prompt will be presented:
-
-```bash
-auth0 test login
-```
-
-### Tail your logs
-
-Once you have a few logins in place, you might want to tail your logs. This is
-done by running the following command:
-
-```bash
-auth0 logs tail
-```
-
-After running that, one might see the following output:
-
-```
-Success Login   9 minutes ago  Username-Password-Authentic...    my awesome app
-```
-
-If there are errors encountered, such as the following example, you may run it
-with the `--debug` flag as follows:
-
-```bash
-auth0 logs tail --debug
-```
+## Customization
 
-The full raw data will be displayed below every error:
+To change the text editor used for editing templates, rules, and actions,
+set the environment variable `EDITOR` to your preferred editor. Example:
 
-```
-Failed Login	hello	7 minutes ago	N/A	my awesome app
-
-	id: "90020210306002808976000921438552554184272624146777636962"
-	logid: "90020210306002808976000921438552554184272624146777636962"
-	date: 2021-03-06T00:28:04.91Z
-	type: f
-	clientid: vXAtoaFdhlmtWjpIrjb9AUnrGEAOH2MM
-	clientname: my awesome app
-	ip: 1.2.3.4
-	description: hello
-	locationinfo: {}
-	details:
-	  body:
-	    action: default
-	    password: '*****'
-	    state: QXV0aDAgaXMgaGlyaW5nISBhdXRoMC5jb20vY2FyZWVycyAK
-	    username: j.doe@gmail.com
-	  connection: Username-Password-Authentication
-	  error:
-	    message: hello
-	    oauthError: access_denied
-	    type: oauth-authorization
-	  qs: {}
-	  session_id: QXV0aDAgaXMgaGlyaW5nISBhdXRoMC5jb20vY2FyZWVycyAK
-	userid: auth0|QXV0aDAgaXMgaGlyaW5nISBhdXRoMC5jb20vY2FyZWVycyAK
+```shell
+export EDITOR="code -w"
+# or
+export EDITOR="nano"
 ```
 
-## Customization
+## Anonymized Analytics Disclosure
 
-To change the text editor used for editing templates, rules, and actions,
-set the environment variable `EDITOR` to your preferred editor:
+Anonymized data points are collected during the use of this CLI. This data includes the CLI version, operating system, timestamp, and other technical details that do not personally identify you.
 
-`export EDITOR="code -w"`
+Auth0 uses this data to better understand the usage of this tool to prioritize the features, enhancements and fixes that matter most to our users.
 
-`export EDITOR="nano"`
+To **opt-out** of this collection, set the `AUTH0_CLI_ANALYTICS` environment variable to `false`.
 
-## Anonymous Analytics
+## Issue Reporting
```

Comment:
Maybe we can make this section similar to everything that goes below this section over here: https://github.com/auth0/auth0-vue#feedback